### ISSUE TITLE
[Windows] open RorConfig crash (latest source code)

### ISSUE THREAD

**Issue by KinG-InFeT** [RigsOfRods/rigs-of-rods#1934]
After compile source (latest), open RorConfig and out this error:
https://imgur.com/a/YNZFINQ

**Comment by AnotherFoxGuy** [comment]
You have to build the zip_and_copy_resources project manually in order to copy the resources required for running RoR

**Comment by KinG-InFeT** [comment]
@AnotherFoxGuy yes done, and open correctly RoRConfig, but on click Save and Play out this error (orher resources missing?)
https://imgur.com/a/NEWWdzw
